addons/api/.github/workflows/ci.yml

```diff
@@ -0,0 +1,76 @@
+name: CI
```

Comment:
What was the purpose of this CI file?

Reply:
Good catch! forgot to delete that one.

Context: this file is included by ember cli when you run the upgrade script. I did not went really deep on this, but when I investigate it, basically they check what CI you use, GH workflows in our case, and adds this file to it.
Supposetly, after adding this file, the CI will run this workflow and basically is running ember-try within the CI.